scripts/merge.sh

```
@@ -540,6 +540,7 @@ function merge_commit {
     else
         staged_with_tab="$(sed 's/^/####\t/' <<< "$2")"
         commitmsg_file=$(mktemp "/tmp/commitmsg.XXXXXX")
+        trap "rm -f '$commitmsg_file'" EXIT INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)